lessons/advanced/gen-stage-1.md

```diff
@@ -0,0 +1,368 @@
+---
```

Comment:
I want to think about this and get some other's feedback.  Right now we don't have any lessons broken into parts and I'm not sure we want to go down that path, we're not writing full fledged tutorials.  If we stick with these lessons functioning as overviews we should be able to get away with two: GenStage and Flow.

I want to pursue example projects but I don't foresee them being in these lessons (at least not directly), I was envisioning those projects either as a new section or a separate practice area of the site.